stories/timeline/props-table.tsx

```diff
@@ -102,16 +102,52 @@ const DATA: ApiTableSectionProps[] = [
     {
         name: "TimelineStatusProps",
         attributes: [
+            {
+                name: "",
+                description: (
+                    <>
+                        <b>DEPRECATED</b>. This component will be replaced
+                        by&nbsp;
+                        <a
+                            href="https://designsystem.life.gov.sg/react/index.html?path=/docs/modules-pill--docs"
+                            rel="noreferrer"
+                            target="_blank"
+                        >
+                            PillProps
+                        </a>
+                    </>
+                ),
+            },
             {
                 name: "type",
                 mandatory: true,
-                description: "The color style of the pill",
+                description: (
+                    <>
+                        The color style of the pill
+                        <br />
+                        <b>DEPRECATED</b>. Use&nbsp;
+                        <a
+                            href="https://designsystem.life.gov.sg/react/index.html?path=/docs/modules-pill--docs"
+                            rel="noreferrer"
+                            target="_blank"
+                        >
+                            PillType&nbsp;
+                        </a>
+                        instead
+                    </>
+                ),
                 propTypes: [`"dark"`, `"light"`],
             },
             {
                 name: "label",
                 mandatory: true,
-                description: "The text to be rendered on the pill",
+                description: (
+                    <>
```

Comment:
Similarly this should still be there but says from v2.4.0 onwards, this attribute will no longer be available